recipes/mesa/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "mesa" %}
+{% set version = "13.0.3" %}
+{% set sha256 = "d9aa8be5c176d00d0cd503cb2f64a5a403ea471ec819c022581414860d7ba40e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: mesa-{{ version }}.tar.xz
+  url: https://mesa.freedesktop.org/archive/{{ version }}/mesa-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+  patches:
+    - llvm-3.9.patch  # [win]
+
+build:
+  number: 0
+  skip: True  # [win and py<35]
+#   skip: True  # [win and not py27]
```

Comment:
Please ignore windows stuff for now, I'm trying to make it work, but there are some problems:

1 - need llvmdev, but it is only available for vc14
2 - need scons, but it only works with python 2.7
3 - on my Windows machine, even after hacking the recipe a bit and being able to install scons and compile with MSVC 2015 (```msvc_compiler: "14.0"  # [win]```), then I get lots of errors from the linker